src/components/Header/Header.tsx

```diff
@@ -13,7 +13,7 @@ export default function Header() {
   const pathname = usePathname();
   // we only change the color when on the home page
   // the osnap page is all light theme
-  const isHomePage = pathname.split("#")[0] === "/";
+  const isHomePage = pathname?.split("#")[0] === "/";
```

Comment:
for some reason typescript is insisting on this now but it didn't before. the reason is that `usePathname` returns null when used in the legacy `pages` directory.